changelogs/current.yaml

```diff
@@ -198,6 +198,11 @@ new_features:
   change: |
     added %RESPONSE_FLAGS_LONG% substitution string, that will output a pascal case string representing the resonse flags.
     The output response flags will correspond with %RESPONSE_FLAGS%, only with a long textual string representation.
+- area: compression
+  change: |
+    added :ref:`remove_accept_encoding_header
+    <envoy_v3_api_msg_extensions.filters.http.compressor.v3.ResponseDirectionOverrides.remove_accept_encoding_header>`
```

Comment:
ResponseDirectionOverrides is a nested message in Compressor. which is giving you the docs error

Reply:
The problem was actually that you need to use `field` not `msg` to refer to fields.  Should now finally be fixed.